custom-metrics-stackdriver-adapter/examples/direct-to-sd/custom-metrics-sd.yaml

```diff
@@ -19,18 +19,23 @@ spec:
       - command:
         - /bin/sh
         - -c
-        - ./sd_dummy_exporter --metric-name=foo --metric-value=40 --pod-id=$(POD_ID)
-        image: gcr.io/google-containers/sd-dummy-exporter:v0.1.0
+        - ./sd_dummy_exporter --metric-name=foo --metric-value=40 --use-new-resource-model --use-old-resource-model --pod-name=$(POD_NAME) --namespace=$(NAMESPACE)
```

Comment:
in case of using old resource model you need to set --pod-id:
https://github.com/GoogleCloudPlatform/k8s-stackdriver/blob/master/custom-metrics-stackdriver-adapter/examples/direct-to-sd/sd_dummy_exporter.go#L71

Reply:
Done